flake8_rst/rst.py

```diff
@@ -37,25 +72,14 @@ def find_sourcecode(src):
         origin_code = match.group('code')
 
         try:
-            min_indent = min(INDENT_RE.findall(origin_code))
+            indent = len(min(INDENT_RE.findall(origin_code)))
         except ValueError:
-            min_indent = ''
-
-        indent = len(min_indent)
-        code = textwrap.dedent(origin_code)
-
-        if '>>>' in code:
-            indent += 4
-            lines = []
-
-            for i, line in enumerate(code.split('\n')):
-                for anchor in ANCHORS:
-                    if line.startswith(anchor):
-                        lines.append(line[len(anchor):])
-                        break
-
-            code = '\n'.join(lines)
+            indent = 0
 
-        line_number = src[:match.start()].count('\n') + match.group('before').count('\n')
+        code = "\n".join([line[indent:] for line in origin_code.split("\n")])
```

Comment:
if rstrip code here, we can simple return `strip_pycon(code, indent, line_number, INTERN_ANCHOR_RE, ANCHOR_IN_RE)`